tests/spec/sort_and_pick/test_sort_extends_to_all_columns_when_underspecified.py

```diff
@@ -0,0 +1,24 @@
+from ..tables import e
+
+title = "Sort extends to all columns when underspecified to ensure that sort order is consistent"
+
+table_data = {
+    e: """
+          |  i1 | i2 |  i3
+        --+----------------
+        1 | 100 |  2 | 101
+        1 | 100 |  1 | 102
+        1 | 100 |  3 | 103
+        2 | 100 |  0 | 500
+        2 | 100 |  1 |   1
+        2 | 101 |  0 |   1
+        """,
+}
+
+
+def test_sorting_extends_to_selected_column(spec_test):
+    spec_test(
+        table_data,
+        e.sort_by(e.i1, e.i2).first_for_patient().i3,
+        {1: 102, 2: 500},
```

Comment:
I can't see where the new behaviour is actually tested here.  The sort order is entirely specified by the values in `i1` and `i2`, isn't it?

Reply:
Yes, you're quite right. This test underwent last-minute revision that has not been kind to it. Thank you.